client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -389,6 +389,17 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 				)
 		);
 
+		const siteTitleFromUrl = ( searchParams.get( 'site_title' ) || '' ).trim();
+		const siteTaglineFromUrl = searchParams.get( 'site_tagline' ) || '';
+
+		// Save site title/description passed to the Assembler.
+		if ( siteTitleFromUrl || siteTaglineFromUrl ) {
+			saveSiteSettings( siteSlugOrId, {
```

Comment:
Originally I chained this to the `setPendingAction()` above, but didn't end up doing it since saving site settings doesn't really depend on the theme activation nor the assembler setup.